experimental/sql/src/impl_to_iterators.py

```diff
@@ -125,8 +220,21 @@ def impl_to_iterators(ctx: MLContext, query: ModuleOp):
       it.SinkOp.get(query.body.blocks[0].ops[0].body.blocks[0].ops[-1]))
   # Adding the return
   query.body.blocks[0].ops[0].body.blocks[0].add_op(Return.get())
-  walker = PatternRewriteWalker(GreedyRewritePatternApplier(
-      [FullTableScanRewriter(), AggregateRewriter()]),
+  index_walker = PatternRewriteWalker(GreedyRewritePatternApplier([
```

Comment:
I need to rewrite all the `IndexByName`s first, since otherwise, their respective operand type does not contain the schema anymore.

Reply:
Makes sense. Add a comment?